src/sisl_toolbox/siesta/minimizer/basis_optimization.py

```diff
@@ -0,0 +1,1031 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
how much of this file is duplicating the yaml conf file? Seems to have some overlap?